src/user/contributing.rst

```diff
@@ -49,4 +49,12 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
 5. Submit a `pull request
    <https://help.github.com/articles/about-pull-requests/>`_ to the main repository proposing your changes.
 
+   Writing guidelines:
+   1. Identify your audience and Understand their skill level.
+   2. Match the technical language with to the audience's skill level proficiency.
+   3. Try to Keep it simple.Incase,if you have to use a lot of complicated terms provide a glossary of key terms
```

Comment:
1. Space after full stop and comma.
2. K in Keep should be in small case.
3. 'In' and 'case' are two words so please consider adding a space between.
4. Also a full stop at the end of this sentence.